assets/webpack.config.js

```diff
@@ -1,10 +1,10 @@
-const path = require('path');
-const glob = require('glob');
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const TerserPlugin = require('terser-webpack-plugin');
-const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
-const CopyWebpackPlugin = require('copy-webpack-plugin');
-const VueLoaderPlugin = require('vue-loader/lib/plugin');
+const path = require("path");
```

Comment:
Was replacing single with double quotation marks done via an automated tool like https://www.npmjs.com/package/prettier?

Reply:
Yes, it's Prettier.  Typically when kicking off projects I'll toss a `.prettierrc` file in the repo to keep us all on the same page formatting wise.  You ok with that?  Just running with Prettier defaults?

Reply:
Absolutely, of course, I just wanted to confirm before creating https://github.com/samvera-labs/digital_collections_elixir_example/issues/30

Reply:
@jrgriffiniii Yea, run with #30  That's super helpful.  Feel free to implement whatever default settings your team is comfortable with.